src/metaschema/oscal_assessment-common_metaschema.xml

```diff
@@ -786,6 +786,72 @@
       </model>
    </define-assembly>
 
+   <define-assembly name="finding">
+      <formal-name>Finding</formal-name>
+      <description>Describes an individual finding.</description>
+      <define-flag name="uuid" required="yes" as-type="uuid">
+        <formal-name>Finding Universally Unique Identifier</formal-name>
+        <!-- Identifier Declaration -->
+        <description>A <a href="/concepts/identifier-use/#machine-oriented">machine-oriented</a>, <a href="/concepts/identifier-use/#globally-unique">globally unique</a> identifier with <a href="/concepts/identifier-use/#cross-instance">cross-instance</a> scope that can be used to reference this finding in <a href="/concepts/identifier-use/#ar-identifiers">this or other OSCAL instances</a>. The locally defined <em>UUID</em> of the <code>finding</code> can be used to reference the data item locally or globally (e.g., in an imported OSCAL instance). This UUID should be assigned <a href="/concepts/identifier-use/#consistency">per-subject</a>, which means it should be consistently used to identify the same subject across revisions of the document.</description>
+      </define-flag>
+      <model>
+        <define-field name="title" min-occurs="1" as-type="markup-line">
+          <formal-name>Finding Title</formal-name>
+          <description>The title for this finding.</description>
+        </define-field>
+        <!-- CHANGE: Added WITH_WRAPPER to description -->
+        <define-field name="description" min-occurs="1" in-xml="WITH_WRAPPER" as-type="markup-multiline">
+          <formal-name>Finding Description</formal-name>
+          <description>A human-readable description of this finding.</description>
+        </define-field>
```

Comment:
Non-blocking: so, did you intend to make this change?

```suggestion
        <field ref="description" in-xml="WITH_WRAPPER" min-occurs="1"/>
```

Reply:
@aj-stein-nist The context of this change was moving the finding assembly from the assessment result model to assessment common so that this assembly could be used in both the assessment result and the poam model without duplication.  Line 803 existed previously, see: https://github.com/usnistgov/OSCAL/blob/develop/src/metaschema/oscal_assessment-results_metaschema.xml#L264. - Did I need to make the change you suggested because of the refactor?  If so, we probably need to revisit the whole section.

Reply:
I guess I was just confused by this line: `<!-- CHANGE: Added WITH_WRAPPER to description -->`. So maybe I misunderstood that? I was interpreting the comment literally, and it is not so much blocking. I thought we might as well address it since it is in the PRed coed, but that referred to something else and not specifically the `description` then?

Reply:
> There are a few "Changed" comments in this assembly from a previous developer. I just moved the assembly verbatim from assessment result.

Sigh, nevermind! You answered the first question I _should have_ asked before embarking done this line of feedback. lol

I am going to resolve this one then.